consensus/polybft/signer/common.go

```diff
@@ -0,0 +1,139 @@
+package bls
+
+import (
+	"crypto/rand"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"math/big"
+
+	bn256 "github.com/umbracle/go-eth-bn256"
+)
+
+var (
+	zeroBigInt = *big.NewInt(0)
+	oneBigInt  = *big.NewInt(1)
+	g2         = *new(bn256.G2).ScalarBaseMult(&oneBigInt)  // Generator point of G2 group
+	zeroG1     = *new(bn256.G1).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+	zeroG2     = *new(bn256.G2).ScalarBaseMult(&zeroBigInt) // Zero point in G2 group
+
+	// this is a copy of point p is a prime over which we form a basic field: 36u⁴+36u³+24u²+6u+1.
+	// TODO delete this point and use bn256.p
```

Comment:
Can you please explain why? Why not do it now?

Reply:
We would have to update the library, and this would eventually be replaced with the edge implementation.